pcdsdevices/epics_motor.py

```diff
@@ -221,6 +186,85 @@ def check_limit_switches(self):
         else:
             return "Low [] High []"
 
+    def get_low_limit(self):
+        """Get the low limit."""
+        return self.low_limit_travel.get()
+
+    def set_low_limit(self, value):
+        """
+        Set the low limit.
+
+        Parameters
+        ----------
+        value : float
+            Limit of travel in the negative direction.
+
+        Raises
+        ------
+        ValueError
+            When motor in motion or position outside of limit.
+        """
+        if not self.moving:
+            # update EPICS
+            _current_high_limit = self._get_epics_limits()[1]
+            if value <= self.position:
+                if value < _current_high_limit:
+                    self.low_limit_travel.put(value)
+                else:
+                    raise ValueError('Could not set motor low limit to %f. '
+                                     'Low limit must be lower than the '
+                                     'current high limit: %f',
+                                     value, _current_high_limit)
+            else:
+                raise ValueError('Could not set motor low limit to %f at '
+                                 'position %f. Low limit must be lower than '
+                                 'the current position.', value, self.position)
+        else:
+            raise ValueError('Could not set the low limit, the motor is in'
```

Comment:
The control flow would be much more obvious if you inverted the `if not self.moving:` and bailed early.
That is, 
```python
if self.moving:
   raise ValueError('...')
# otherwise, do this:
```

Reply:
Thanks - i have changed this in b3a7c93

Reply:
Nice - so much cleaner, don't you think? 👍

Reply:
Yes of course, thank you 😊